content/api/commands/session.md

```diff
@@ -146,13 +145,11 @@ Cypress.Commands.add('login', (username, password) => {
 ```javascript
 Cypress.Commands.add('login', (username, password) => {
   cy.session([username, password], () => {
```

Comment:
any ID with username & password should be removed from the examples IMO. We dont' have a log:false option and we don't want to direct users to use sensitive information which they may not want video recorded/face-up.

Reply:
This is a legitimate concern but we currently do this absolutely everywhere - guides, API docs, recipes, RWA, Kitchen Sink etc etc. If we're going to ban this practice there needs to be a larger conversation about that.

Reply:
We don't need to ban it, but we should remove it from our examples to provide good guidance. It is something we need to be updating in our documentation and examples.